srcs/client/src/Pages/Chat/Chat.tsx

```diff
@@ -77,6 +77,7 @@ export default function Chat(props: any) {
       changeChannelPassword(currentChannel.id, newChannelPassword);
     } else {
       setChannelPassword(currentChannel.id, newChannelPassword);
+      handleClosePassworChannel();
```

Comment:
There is a typo Passwor / Password but it is the case also in the function declaration so it is ok :)